### DISCUSSION TITLE
Marketing Proposal for 6 months of increased engagement for Darwinia

### DISCUSSION THREAD

**Discussion by cryptohustler101** [darwinia-network/darwinia#1591] [General]
Hello, Dustin Lee here—long-term $RING holder, early mod of Evolution Land, and a dedicated ambassador for the Polkadot ecosystem and Darwinia. I've been advocating for Darwinia for years and worked with Evolution Land for 6 months. With the current market, Darwinia has a great opportunity for price appreciation, and I’d like to propose a marketing budget to further boost engagement through my agency, A1 Marketing Bros.

Proposal Overview:

1.5M $RING total @ $0.00174 to cover a 6-month runway (spending 1M RING ~$72.58/week over 24 weeks for marketing, liquidating only as needed).

500k $RING will cover my labor.

Focus Areas:

X (formerly Twitter):
Boost engagement and followers on relevant accounts and Posts pertaining to (Helix, Snowswap, RingDAO, DCDAO, Darwinia, EvolutionLand, Subscan).

YouTube:
Increase views, likes, and subscribers for Darwinia’s main channel.

We've had success running a trial with Partnership Dude (David) and I have recently boosted engagement for Helix, DCDAO, Snowswap, and RingDAO on various posts on X. This proposal builds on that, aiming to expand Darwinia's reach.

I'm confident this will help us grow Darwinia further! Thank you for considering this proposal.

**Comment by hackfisher** [comment]
Since the market and community are currently mainly managed by DCDAO, I suggest collaborating with DCDAO's existing plans, as referenced here: https://github.com/orgs/dcdao/discussions/169.

Additionally, please refer to DCDAO's budget and Cap. It is not recommended to incur extra expenses to avoid putting pressure on liquidity and market price due to further outflow of RING.

Ref: https://github.com/orgs/dcdao/discussions/156

**Reply by cryptohustler101** [reply]
Thanks for sharing the insights and the references! Collaborating with DCDAO's existing plans makes sense, especially to align with their budget and cap. I agree that we should avoid extra expenses to maintain liquidity and avoid affecting the market price of RING.

I'll review the discussions you've linked, and we can look into how best to integrate our efforts with DCDAO's strategy.

Appreciate your guidance on this!
